Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:24.04
```

Comment:
Latest LTS version

Reply:
I'd like to avoid slim here because we're actually configuring a bit more in this image, namely timezones and adding some tooling. That tooling was from 3 years ago, and it'd be fine to drop now, I think. "Porting" this to a -bookworm or similar python version would be cake though, because those are Debian builds and ubuntu is just a downstream tweak of Debian. I also know the Debian ecosystem backwards and forwards -- can't say that about alpine in slim builds.

But - yea, I'd be happy to port this into a python focused image -- it's a good idea, and i'll do it. If it's anything more than a real-quick-edit, i'll pen a super short issue, give it a point or two, and do our thing more formally.